apps/wallet/src/ui/app/components/sui-apps/index.tsx

```diff
@@ -21,17 +21,17 @@ function AppsPlayGround() {
 	const ecosystemApps = useFeature<DAppEntry[]>(FEATURES.WALLET_DAPPS).value;
 	const location = useLocation();
 
-	const queryParams = new URLSearchParams(location.search);
-	const tagFilter = queryParams.get('tagFilter');
+	const pathSegments = location.pathname.split('/');
```

Comment:
You should be able to name this param in the route definition instead of doing this work manually.